clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -102,6 +102,9 @@
     public static final String ENABLE_STICKY_METADATA_FETCH_DOC = "Fetch metadata from the least loaded broker if false. Otherwise fetch metadata "
                                                          + "from the same broker until it is disconnected.";
 
+    public static final String ENABLE_CLIENT_RESPONSE_LEAK_CHECK = "enable.client.resonse.leakcheck";
```

Comment:
to help distinguish linkedin features from vanilla ones, and avoid conflicts, we like to make configs keys for our features start with "linkedin". so - "linkedin.enable.client.response.leakcheck"